spec/section_spec.rb

```diff
@@ -4,37 +4,56 @@
 
 describe SitePrism::Page do
   describe '.section' do
+    class Page < SitePrism::Page; end
+    class Section < SitePrism::Section; end
+
+    class PageWithAnonymousSection < SitePrism::Page
+      section :anonymous_section, '.section' do |s|
+        s.element :title, 'h1'
+      end
+    end
+
+    class PageWithSection < SitePrism::Page
+      section :section, Section, '.section'
+    end
+
+    context 'second argument is a Class' do
+      subject { PageWithSection.new }
+
+      it 'should create a section using the Class' do
+        expect(subject).to respond_to(:section)
```

Comment:
I think unnamed subjects are fine if they're being declared close to the assertion. But if there's a lot of distance between the subject declaration and the assertion, it's a readability problem because it's no longer easy to know what `subject` means.

But also, in this case you don't actually need an explicit subject declaration so I'd agree with just using a simple `it` block as @tgaff suggests.